log-viewer/resources/css/TreeView.css

```diff
@@ -28,6 +28,21 @@
   color: red;
   margin-right: 5px;
 }
+#breadcrumb {
```

Comment:
looks like this did not get formatted?
is your ide ignoring editorconfig? do you have prettier running?

Reply:
As far as I can tell, prettier is running - but it doesn't seem interested in the whitespace...
I've converted to spaces manually.